src/components/article/AllArticles.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getAllArticles } from '../../actions/articles';
+import './AllArticles.scss';
+
+class AllArticles extends Component {
+  state = {
+    done: false,
+    defaultImg: 'https://tmsnrt.rs/2FNDBT2',
+    defaultText: 'Just a placeholder',
+  };
+
+  async componentDidMount() {
+    const { getAllArticles: getAll } = this.props;
+    await getAll();
+    this.setState({ done: true });
+  }
+
+  handleClick = async () => {
+    const { getAllArticles: getAll } = this.props;
+    await getAll();
+    this.setState({ done: true });
+  };
+
+  renderArticles() {
+    const { allArticles } = this.props;
+    const { done, defaultImg, defaultText } = this.state;
+    if (done) {
+      return allArticles.allArticles.map(article => (
+        <div className="col s12 m6 l6" key={article.id}>
+          <div className="card">
+            <div className="card-image">
+              <img
+                src={article.imgUrl ? article.imgUrl : defaultImg}
+                alt={article.title ? article.title : defaultText}
+              />
+            </div>
+
+            <div className="card-content">
+              <span className="card-title">{article.title}</span>
+              <p className="article-description">{article.description}</p>
+
+              <div className="article-author">
+                <span>by</span>
+                <b>{article.author.username}</b>
+              </div>
+
+              <div className="article-tags">
+                {article.tags.map((tag, i) => (
+                  <span className="tags" key={tag[i]}>
+                    {tag}
+                  </span>
+                ))}
+              </div>
+            </div>
+          </div>
+        </div>
+      ));
+    }
+    return null;
+  }
+
+  render() {
+    return (
+      <div>
+        <div className="row">
+          <div className="col s12">
+            <h6 className="trending-tags">Trending tags goes here</h6>
+          </div>
+        </div>
+        <div className="row feature">
+          <div className="col s12 feature-image">
+            <h1 className="feature-text">
+              <p>
+                {
+                  'The more that you read, the more things you will know. The more you learn, the more places you will go'
+                }
+                - Dr. Suess,
+              </p>
+            </h1>
+          </div>
+        </div>
+        {/* <div className="container"> */}
```

Comment:
Please take out this comment if this line is no longer being utilized

Reply:
Thanks for the observation. I have removed it.